cmd/go-camo/main.go

```diff
@@ -227,6 +228,7 @@ func (cli *CLI) Run() {
 	config.MaxSize = cli.MaxSize * 1024
 	config.RequestTimeout = cli.ReqTimeout
 	config.MaxRedirects = cli.MaxRedirects
+        config.MaxSizeRedirect = cli.MaxSizeRedirect
```

Comment:
This uses spaces instead of tabs for indentation.

Reply:
also fixed in HEAD. 👍